starter-code/styles/modules.css

```diff
@@ -9,13 +9,16 @@
 }
 
 /* TODO: Add styles for the icon-menu class. It should be bigger, and off to the right. */
+#hamburger{
+  float: right;
+  font-size: 2.5em;
+}
```

Comment:
I'd prefer to see the css styled using classes rather than id's, here we could simply use .icon-menu, assuming we wont be using that icon again. No need to fix it up in this assignment, but watch for that going forward.